build_tools/docker/bazel-tensorflow-vulkan/Dockerfile

```diff
@@ -17,19 +17,15 @@
 
 FROM gcr.io/iree-oss/bazel-tensorflow AS final
 
-RUN apt-get update && apt-get install -y wget
+ARG VULKAN_SDK_VERSION=1.2.154.0
```

Comment:
Kind of sucks that we have to have this stay in sync with the base vulkan image. It appears there's no way to just copy an env variable from another image (boooo https://github.com/moby/moby/issues/37345).

Does it work to have a RUN command that sets path? That would allow us to 

```dockerfile
# vulkan/Dockerfile
RUN echo "/opt/vulkan-sdk/${VULKAN_SDK_VERSION}/x86_64" > /vulkan_path
```

```dockerfile
# bazel-tensorflow-nvidia/Dockerfile
RUN export PATH="$(cat /vulkan_path)/bin:$PATH"
```

I think it's not, and we'd have to actually use an ENTRYPOINT script to set PATH and then execute the provided command.

Reply:
Yeah. It's a bit annoying but we can start with this. I'll leave further docker image re-architecture to you. :)

Reply:
Can you add comments to all the places where the vulkan version is configured that it needs to be kept in sync with the others?